data/schemas/platform-detect.schema

```diff
@@ -0,0 +1,54 @@
+{
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "description": "Soletta's platform detection file",
+
+  "definitions": {
+    "regex_array": {
+      "type": "array",
+      "minItems": 1,
+      "uniqueItems": true,
+      "items": { "type": "string" }
+    },
+
+    "rule": {
+      "type": "object",
+      "additionalProperties": false,
```

Comment:
It follows http://json-schema.org/latest/json-schema-validation.html#anchor65. I didn't find "additional_properties" among json-schema docs.